packages/login/network/walletProvider.js

```diff
@@ -3,40 +3,14 @@
  * Licensed under the AGPL Version 3 license.
  */
 
-import bitcoin from 'bitcoinjs-lib'
-import * as WavesApi from '@waves/waves-api'
-import nemSdk from 'nem-sdk'
-import { byEthereumNetwork } from './NetworkProvider'
-import { createBCCEngine, createBTCEngine, createLTCEngine, createBTGEngine } from './BitcoinUtils'
-import EthereumEngine from './EthereumEngine'
-import { createNEMEngine } from './NemUtils'
-import { createWAVESEngine } from './WavesUtils'
-import NemWallet from './NemWallet'
-import WavesWallet from './WavesWallet'
 import EthereumWallet from './EthereumWallet'
+import privateKeyProvider from './privateKeyProvider'
 
 class WalletProvider {
-  getProvider (walletJson, password, { url, network } = {}) {
-    const networkCode = byEthereumNetwork(network)
+  getProvider (walletJson, password, settings = {}) {
```

Comment:
If we have only one provider here and file packages/login/network/walletProvider.js used only in one place ( packages/login/redux/network/thunks.js ), maybe it is better move 'getProvider' to the file packages/login/redux/network/utils.js (with appropriate imports) and delete 'packages/login/network/walletProvider.js' at all? It is not a common file anymore.